### PR TITLE
Updated migration doc to include additional findings.

### PR DESCRIPTION
**Pull request by dasien**
## Fixes

- Fixes #_____

## Proposed Changes/Todos

- [ ] Todo 1

## Pull Request checklist:

- [ ] I've named my PR in the form of "Fixes #issue. Terse description."
- [ ] My code follows the [style guidelines of Terminal.Gui](https://github.com/gui-cs/Terminal.Gui/blob/develop/.editorconfig) - if you use Visual Studio, hit `CTRL-K-D` to automatically reformat your files before committing.
- [ ] My code follows the [Terminal.Gui library design guidelines](https://github.com/gui-cs/Terminal.Gui/blob/develop/CONTRIBUTING.md)
- [ ] I ran `dotnet test` before commit
- [ ] I have made corresponding changes to the API documentation (using `///` style comments)
- [ ] My changes generate no new warnings
- [ ] I have checked my code and corrected any poor grammar or misspellings
- [ ] I conducted basic QA to assure all features are working

### CODE REVIEW

File: docfx/docs/migratingfromv1.md [hunk 1]
```diff
@@ -204,6 +218,21 @@ The cursor and focus system has been redesigned in v2 to be more consistent and
 * Set [View.CursorVisibility](~/api/Terminal.Gui.View.CursorVisibility.yml) to the cursor style you want to use.
 * Remove any overrides of `OnEnter` and `OnLeave` that explicitly change the cursor.
 
+## Button.Clicked Event Renamed
+
+The `Button.Clicked` event has been renamed `Button.MouseClick`
```
Comment (tznind) [inline docfx/docs/migratingfromv1.md:223]: I think this should be the `Accept` event right @BDisp?

In v1 Clicked was fired under both Keyboard Enter and Mouse Click.  So closest direct behavioural replacement for `Button.Clicked` would be to use the `Button.Accept` event.

Comment (BDisp) [reply docfx/docs/migratingfromv1.md:223]: Correct. It's using the `View.Accept` event.

https://github.com/gui-cs/Terminal.Gui/blob/a661fcecf7e60b694649a4d93df781c7ef5952ee/Terminal.Gui/View/View.cs#L115

File: docfx/docs/migratingfromv1.md [hunk 2]
```diff
@@ -236,8 +265,9 @@ If you previously had a lamda expression, you can simply add the extra arguments
 
 ```diff
 - btnLogin.Clicked += () => { /*do something*/ };
-+ btnLogin.Clicked += (s,e) => { /*do something*/ };
++ btnLogin.MouseClick += (s,e) => { /*do something*/ };
```
Comment (tig) [inline docfx/docs/migratingfromv1.md:268]: The correct event is `Accept` in v2.

If a dev really wants to know that the button was acted upon by the user *with the mouse* (which would be a very bespoke case), the dev can subscribe to `MouseClick`.

File: docfx/docs/migratingfromv1.md [hunk 3]
```diff
@@ -175,6 +176,12 @@ The API for handling keyboard input is significantly improved. See [Keyboard API
 * Use [View.Keybindings](~/api/Terminal.Gui.View.Keybindings.yml) to configure key bindings to `Command`s.
 * It should be very uncommon for v2 code to override `OnKeyPressed` etc... 
 * Anywhere `Ctrl+Q` was hard-coded as the "quit key", replace with `Application.QuitKey`.
+* Replace `Application.RootKeyEvent` with `Application.KeyDown`.
```
Comment (tig) [inline docfx/docs/migratingfromv1.md:179]: Please add to the end of this bullet: "If the reason for subscribing to `RootKeyEvent` was to enable an application-wide action based on a key-press, consider using `Application.KeyBindings` instead.

**Review by tig — CHANGES_REQUESTED** [review]

**Review by tig — APPROVED** [review]

**Pull request merged**